src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const currentObject = { ...state };
```

Comment:
```suggestion
  const stateCopy = { ...state };
```
Actually, it's not a currentObject